components/camel-kamelet/pom.xml

```diff
@@ -43,6 +43,54 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-core-model</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I choose to set both, components and 3rd party dependency as provided, because they should be declared in the Kamelet YAML using those actions. If you have other idea let me know.